### PR TITLE
Update wslu install pitfalls

### PR DESCRIPTION
**Pull request by ohjuny**
Based on Piazza post [@432](https://piazza.com/class/lbnv6k5pd0y6bz/post/432).
From [this link](https://wslutiliti.es/wslu/install.html#ubuntu), it appears some versions of Ubuntu require a different way of installing wslu (for wslview). The added commands are also taken from the link.
Unfortunately I do not have a Windows machine so am unable to test this, but it seemed to have worked for the student who posted on Piazza.

### CODE REVIEW

File: docs/cli.md
Comment (awdeorio) [inline docs/cli.md:261]: I'm a little confused by the multiple install commands could you help me understand?

Comment (ohjuny) [reply docs/cli.md:261]: To be completely honest, I do not exactly know what is happening here; the commands are taken straight from the [wslu wiki](https://wslutiliti.es/wslu/install.html#ubuntu). It seemed to work for the student on [Piazza](https://piazza.com/class/lbnv6k5pd0y6bz/post/432).

Comment (ohjuny) [reply docs/cli.md:261]: Sounds good. We could also put this on hold to see if a decent number of students are facing the install issue. So far it does just seem like the one student on Piazza.

Comment (awdeorio) [reply docs/cli.md:261]: I think the best next step would be to try this on Ubuntu-20.04 and Ubuntu-22.04.  I suspect that is the difference.  You could do it with virtualbox / vagrant.

Comment (awdeorio) [reply docs/cli.md:261]: Looks like `wslu` might be installed by default on Ubuntu 20.04 WSL and not on Ubuntu 22.04 WSL. https://askubuntu.com/questions/1409481/why-does-xdg-open-use-wslview#comment2448974_1409481

Comment (awdeorio) [reply docs/cli.md]: I took a shot at it.  Just got somebody to check the output of `lsb_release`, which I guessed.

**Review by awdeorio — APPROVED** [review]
LGTM after checking the output of `lsb_release`

**Comment by ohjuny** [comment]
I do not have merging permissions, do I believe you may have to merge this!

**Pull request merged**
